src/utils.js

```diff
@@ -127,25 +127,62 @@ exports.hideAbsolutePathsInObject = (object) => {
   }
 
   if (_.isString(object)) {
-    const words = object.split(' ').map(word => (path.isAbsolute(word) ? path.basename(word) : word))
-    return words.join(' ')
+    const words = object.split(' ')
+    .map(word => (path.isAbsolute(word) ? path.basename(word) : word))
+    .join(' ')
+    const anonymizedWords = exports.anonymizePaths(words)
+    return anonymizedWords
   }
 
   // Avoid "Maximum call stack size exceeded" errors
   object = jc.decycle(object)
 
-  return _.deepMapValues(object, (value) => {
-    if (!_.isString(value)) {
-      return value
+  return _.deepMapValues(object, exports.anonymizePaths)
+}
+
+/**
+ * @summary Anonymize paths in logs
+ * @function
+ * @public
+ *
+ * @param {String|Object} value - value to anonymize
+ * @returns {String|Object} transformed value
+ */
+exports.anonymizePaths = (value) => {
+  if (!_.isString(value)) {
+    return value
+  }
+  // Don't alter disk devices, even though they appear as full paths
+  if (value.startsWith('/dev/') || value.startsWith('\\\\.\\')) {
+    return value
+  }
+
+  const valueArray = value.split('\n')
+  const anonymizedValue = valueArray.map((line) => {
+    // Check for the file:// protocol
+    const fileRegex = /file:\/\//i
+    if (fileRegex.test(line).valueOf()) {
+      return `${_.first(line.split('file://'))}${_.last(line.split('/'))}`
```

Comment:
There could be more than one path on a line.

Reply:
Not for this specific case, this is for the error name & message that displays on Sentry so it's filtered to display just the file name (in case there is one)